### PR TITLE
(DOCSP-16849) Backfills docs content for organizations commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for organizations commands. Also addresses one ticket to add more detail to liveMigration commands (but doesn't perform complete backfill/rewrites for those commands).

_Jira ticket:_

https://jira.mongodb.org/browse/DOCSP-16849
https://jira.mongodb.org/browse/DOCSP-16850
https://jira.mongodb.org/browse/DOCSP-16851
https://jira.mongodb.org/browse/DOCSP-16852
https://jira.mongodb.org/browse/DOCSP-16853
https://jira.mongodb.org/browse/DOCSP-27330

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/iam/organizations/delete.go [hunk 1]
```diff
@@ -51,11 +52,14 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <ID>",
 		Aliases: []string{"rm"},
-		Short:   "Delete an organization.",
+		Short:   "Remove the specified organization.",
+		Long:    "Organizations with active projects can't be removed.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"IDDesc": "Organization identifier.",
+			"IDDesc": "Unique 24-digit string that identifies the organization.",
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/delete.go:59]:
```suggestion
			"IDDesc": "Unique 24-hexadecimal digit string that identifies the organization.",
```

File: internal/cli/iam/organizations/describe.go [hunk 2]
```diff
@@ -61,10 +62,12 @@ func DescribeBuilder() *cobra.Command {
 		Use:     "describe <ID>",
 		Aliases: []string{"show"},
 		Args:    require.ExactArgs(1),
-		Short:   "Describe an organizations.",
+		Short:   "Return the details for the specified organizations.",
 		Annotations: map[string]string{
-			"IDDesc": "Organization identifier.",
+			"IDDesc": "Unique 24-digit string that identifies the organization.",
```
Comment (kanchana-mongodb) [inline internal/cli/iam/organizations/describe.go:67]:
```suggestion
			"IDDesc": "Unique 24-hexadecimal digit string that identifies the organization.",
```

**Review by kanchana-mongodb — APPROVED** [review]
LGTM with some non-blocking suggestions.

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**